services/cache/cache.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package cache
+
+import (
+	"time"
+)
+
+// Cache is a representation of any cache store that has keys and values
+type Cache interface {
+	Purge()
```

Comment:
While we are here, I would also request to add godocs for each of the exported methods. This helps someone reading the interface and trying to get an idea of what it does.

Reply:
Done!